annotator/src/main/java/org/cbioportal/models/MutationRecord.java

```diff
@@ -0,0 +1,495 @@
+    }
+    
+    public String getSequence_Source() {
+        return this.sequencingSource == null ? "" : this.sequencingSource;
+    }
+    
+    public void setSequence_Source(String sequencingSource) {
+        this.sequencingSource = sequencingSource;
+    }
+    
+    public String getValidation_Method() {
+        return this.validationMethod == null ? "" : validationMethod;
+    }
+ 
+    public void setValidation_Method(String validationMethod) {
+        this.validationMethod = validationMethod;
+    }
+    
+    public String getScore() {
+        return this.score == null ? "" : this.score;
+    }
+    
+    public void setScore(String score) {
+        this.score = score;
+    }
+    
+    public String getBAM_File() {
+        return this.bamFile == null ? "" : this.bamFile;
+    }
+    
+    public void setBAM_File(String bamFile) {
+        this.bamFile = bamFile;
+    }
+    
+    public String getSequencer() {
+        return this.sequencer == null ? "" : this.sequencer;
+    }
+    
+    public void setSequencer(String sequencer) {
+        this.sequencer = sequencer;
+    }
+    
+    public String getTumor_Sample_UUID() {
+        return this.tumorSampleUUID == null ? "" : this.tumorSampleUUID;
+    }
+    
+    public void setTumor_Sample_UUID(String tumorSampleUUID) {
+        this.tumorSampleUUID = tumorSampleUUID;
+    }
+    
+    public String getMatched_Norm_Sample_UUID() {
+        return this.matchedNormSampleUUID == null ? "" : this.matchedNormSampleUUID;
+    }
+    
+    public void setMatched_Norm_Sample_UUID(String matchedNormSampleUUID) {
+        this.matchedNormSampleUUID = matchedNormSampleUUID;
+    }
+    
+    public String gett_ref_count() {
+        return this.tRefCount == null ? "" : this.tRefCount;
+    }
+    
+    public void sett_ref_count(String tRefCount) {
+        this.tRefCount = tRefCount;
+    }
+    
+    public String gett_alt_count() {
+        return this.tAltCount == null ? "" : this.tAltCount;
+    }
+    
+    public void sett_alt_count(String tAltCount) {
+        this.tAltCount = tAltCount;
+    }
+    
+    public String getn_ref_count() {
+        return this.nRefCount == null ? "" : this.nRefCount;
+    }
+    
+    public void setn_ref_count(String nRefCount) {
+        this.nRefCount = nRefCount;
+    }
+    
+    public String getn_alt_count() {
+        return this.nAltCount == null ? "" : this.nAltCount;
+    }
+    
+    public void setn_alt_count(String nAltCount) {
+        this.nAltCount = nAltCount;
+    } 
+    
+    public void addAdditionalProperty(String property, String value) {
+        this.additionalProperties.put(property, value);
+    }
+    
+    public Map<String, String> getAdditionalProperties() {
+        return this.additionalProperties;
+    }
+    
+    public void setAdditionalProperties(Map<String, String> additionalProperties) {
+        this.additionalProperties = additionalProperties;
+    } 
+    
+    public List<String> getFieldNames() {
```

Comment:
This is minor, but whats the point of these methods (same one in AnnotatedRecord)?  Why not hardcode a member list in a static function instead of recomputing for each call and just have getHeader() as part of the "public" api?